source/sass/blocks/logo.scss

```
@@ -0,0 +1,3 @@
+.logo {
+  background-color: rgba($color: #000000, $alpha: 0.3);
```

Можно еще попроще делать:
```suggestion
  background-color: rgba(#000000, 0.3);
```

А еще можно использовать переменную, например так (ее конечно создать надо еще):
```suggestion
  background-color: rgba($color-black, 0.3);
```